test/e2e/common/node/kubelet.go

```diff
@@ -143,48 +144,32 @@ var _ = SIGDescribe("Kubelet", func() {
 			Release: v1.13
 			Testname: Kubelet, hostAliases
 			Description: Create a Pod with hostAliases and a container with command to output /etc/hosts entries. Pod's logs MUST have matching entries of specified hostAliases to the output of /etc/hosts entries.
-			Kubernetes mounts the /etc/hosts file into its containers, however, mounting individual files is not supported on Windows Containers. For this reason, this test is marked LinuxOnly.
 		*/
-		framework.ConformanceIt("should write entries to /etc/hosts [LinuxOnly] [NodeConformance]", func() {
-			podClient.CreateSync(&v1.Pod{
-				ObjectMeta: metav1.ObjectMeta{
-					Name: podName,
-				},
-				Spec: v1.PodSpec{
-					// Don't restart the Pod since it is expected to exit
-					RestartPolicy: v1.RestartPolicyNever,
-					Containers: []v1.Container{
-						{
-							Image:   framework.BusyBoxImage,
-							Name:    podName,
-							Command: []string{"/bin/sh", "-c", "cat /etc/hosts; sleep 6000"},
-						},
-					},
-					HostAliases: []v1.HostAlias{
-						{
-							IP:        "123.45.67.89",
-							Hostnames: []string{"foo", "bar"},
-						},
-					},
+		framework.ConformanceIt("should write entries to /etc/hosts [NodeConformance]", func() {
+			pod := e2epod.NewAgnhostPod(f.Namespace.Name, podName, nil, nil, nil, "etc-hosts")
```

Comment:
For this change, `when scheduling a busybox` of line 140 needs to be `when scheduling an agnohost`.